croptalk/document_retriever.py

```diff
@@ -75,17 +79,36 @@ def get_documents(
         return formatted_docs
 
     @staticmethod
-    def _format_query_response(query_response: QueryReturn) -> List[str]:
+    def read_pdf_from_s3(bucket_name, file_key):
```

Comment:
Victor found out that a significant proportion of SP docs are just dots, or spaces, which eat up tokens. Should we make an option to clean up the text?

Reply:
ok good point! I'll create a function to clean it up

Reply:
Awesome thanks!

Reply:
@katerina-cropguard @VictorChabot I will create a subsequent PR for this feature.

Reply:
@katerina-cropguard @VictorChabot so looking at the excel sheet, most "points" that should be removed are affecting our chunks are CIH documents. I can create a filter to remove them but I think the overall quality of the tool would be improved if we remove those weird occurrences when we generate chunks upstream.